### PR TITLE
Bookswagon site integration changes

### PR DESCRIPTION
**Pull request by Naveenkhasyap**
this is the PR for  Bookswagon site integration [Issue]( https://github.com/vshelke/bookpro/issues/44)

### CODE REVIEW

File: bookpro/find_books/integrations/bookswagon.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        if self.check(title):
+            data['title'] = title.text
+        if(self.check(author)):
+            data['author'] = author.text
+        if(self.check(link)):
+            data['link'] = link
+        if(self.check(price)):
+            data['price'] = float(price.text[3:].replace(',', ''))
+        if(self.check(img)):
+            r = requests.head(img['src'])
```
Comment (vshelke) [inline bookpro/find_books/integrations/bookswagon.py]: Making a request for every book will increase the latency for this thread. Can we think of an alternative which does not include making an request for each product. Also, it is only checking the 200 status so, we can ignore checking the head and directly use the `img['src']`.

File: bookpro/find_books/integrations/bookswagon.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+        return data
+
+    def run(self):
+        url = 'https://www.bookswagon.com/search-books/'+self.query
+        r = requests.get(url)
+        soup = BeautifulSoup(r.text, 'html.parser')
+        products = soup.findAll(
+            "div", {"class": "list-view-books"})
+        print(products)
```
Comment (vshelke) [inline bookpro/find_books/integrations/bookswagon.py]: The print statement needs to be removed. 😋

File: bookpro/find_books/integrations/bookswagon.py (hunk 1 above)
Comment (Naveenkhasyap) [reply bookpro/find_books/integrations/bookswagon.py]: issue here is some case it gives url for image src but the url does not contain any image in it hence this head check and assign default image not available url.

File: bookpro/find_books/integrations/bookswagon.py (hunk 2 above)
Comment (Naveenkhasyap) [reply bookpro/find_books/integrations/bookswagon.py]: taken care 👍

File: bookpro/find_books/integrations/bookswagon.py (hunk 1 above)
Comment (vshelke) [reply bookpro/find_books/integrations/bookswagon.py]: Oh okay then this is good to go. 👍

**Comment by vshelke** [comment]
@Naveenkhasyap Thank you for your contribution. Will merge the PR now.

**Pull request merged**